test/OpenTelemetry.Tests.Stress/Skeleton.cs

```diff
@@ -145,10 +181,13 @@ public static void Stress(int concurrency = 0, int prometheusPort = 0)
         var cntCpuCyclesTotal = GetCpuCycles();
         var cpuCyclesPerLoopTotal = cntLoopsTotal == 0 ? 0 : cntCpuCyclesTotal / cntLoopsTotal;
         Console.WriteLine("Stopping the stress test...");
-        Console.WriteLine($"* Total Runaway Time (seconds) {watchForTotal.Elapsed.TotalSeconds:n0}");
+        Console.WriteLine($"* Total Runway Time (Seconds) {watchForTotal.Elapsed.TotalSeconds:n0}");
```

Comment:
Update the README file as well.

Reply:
Forgot to ask, was "Runaway" intentional? In the running output we have "Runway" so it seemed like a typo.

Reply:
I don't know what's the perfect term, I just picked a term that seems to fit the intention http://osr507doc.xinuos.com/en/HANDBOOK/runaway_proc.html. (e.g. if user keeps this process, it'll consume lots of system resources)

The comment here is more emphasizing on "whatever term we choose, we have to be consistent".